Tests/test_Nexus.py

```diff
@@ -363,6 +363,47 @@ def test_NexusTest1(self):
 ;
 """)  # noqa : W291
 
+    @staticmethod
+    def test_write_alignment():
+        # Default causes no interleave (columns <= 1000)
+        records = [SeqRecord(Seq("ATGCTGCTGA" * 90, alphabet=ambiguous_dna), id=_id) for _id in ["foo", "bar", "baz"]]
+        a = MultipleSeqAlignment(records, alphabet=ambiguous_dna)
+
+        handle = StringIO()
+        NexusWriter(handle).write_alignment(a)
+        handle.seek(0)
+        data = handle.read()
+        assert "ATGCTGCTGA" * 90 in data
```

Comment:
Can you switch that ``assert`` statement (and the similar ones below) to:

```python
self.assertIn("ATGCTGCTGA" * 90, data)
```

This is new in Python 2.7, prior to that we'd have used a workaround like ``self.assertTrue("ATGCTGCTGA" * 90 in data)`` instead.